db/migrations/sqls/20200601143136-eligibility-criteria-up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE eligibility_criteria (
+    id serial PRIMARY KEY,
+    grant_application_id INTEGER REFERENCES grant_application(id),
+    trading_in_hackney BOOLEAN NOT NULL,
+    small_micro_business BOOLEAN NOT NULL,
+    trading_on_20200311 BOOLEAN NOT NULL,
```

Comment:
What does the number in this variable name mean?

Reply:
The question answered is `Was your business trading on the 11th March 2020?`, I went for the date in ISO style.